erikbenedict/hype-hub
# Feature/achievement routes

## TEST

- Sign up as a new user to test register page
- On dashboard click on all links to ensure functionality
- On dashboard create new topic to see that modal functions properly, and a card is rendered with that title
- Refresh the page to see card persist
- Click on the Topic card to be redirected to topic page
- On topic page click on all links to ensure functionality
- On topic page create new achievement to see that modal functions properly and a card is rendered with the user input
- Refresh the page to see card persist
- Log in as a different user and create a topic with the same name
- Click on that topic to see that other user's achievement aren't listed
